Libraries/LibJS/Runtime/ObjectConstructor.cpp

```diff
@@ -146,4 +147,19 @@ Value ObjectConstructor::define_property(Interpreter& interpreter)
     return &object;
 }
 
+Value ObjectConstructor::is(Interpreter& interpreter)
+{
+    auto value1 = interpreter.argument(0);
+    auto value2 = interpreter.argument(1);
+    if (value1.is_nan() && value2.is_nan())
+        return Value(true);
+    if (value1.is_number() && value1.as_double() == 0 && value2.is_number() && value2.as_double() == 0) {
+        // check if both are +0 or -0
+        auto value1_is_positive_zero = 1.0 / value1.as_double() == __builtin_huge_val();
+        auto value2_is_positive_zero = 1.0 / value2.as_double() == __builtin_huge_val();
```

Comment:
Maybe these would make sense as helpers on `Value`? I imagine we'll want to check if a `Value` is `+0` or `-0` somewhere else at some point..

Reply:
Definitely! Done :)